app/src/main/java/com/example/gymrats/Post.java

```diff
@@ -0,0 +1,77 @@
+package com.example.gymrats;
+
+
+import android.util.Log;
+
+import com.parse.ParseClassName;
+import com.parse.ParseFile;
+import com.parse.ParseObject;
+import com.parse.ParseUser;
+
+import java.util.Date;
+
+@ParseClassName("Post")
+public class Post extends ParseObject {
```

Comment:
do we assume there is only one description/image/user in the database?

Reply:
i see this is for post class, does the database use a post java object as some sort of key?

Reply:
yeah i used the post class to get the key for the required from the database.